docs/operators/rest.md

```diff
@@ -0,0 +1,76 @@
+---
+title: REST on GraphQL
+description: Exposing REST endpoints on top of GraphQL via @rest directive.
+---
+
+Tailcall's `@rest` directive exposes standard REST endpoints, enabling the fetching of data from REST APIs directly through GraphQL queries.
+
+This guide will show you how to use the `@rest` directive to create a RESTful service over a GraphQL layer for retrieving product information.
+
+## The @rest Directive Basics
+
+- **method**: Specifies the HTTP method (GET, POST, etc.).
+
+- **path**: Defines the endpoint URL, supporting dynamic values from query arguments.
+
+## Example: Retrieving Product Information
+
+![Diagram](/images/docs/rest-diagram.svg)
```

Comment:
The image is great but should explain what's happening here. To summarize, a REST route is created dynamically from the provided GraphQL query/mutation. Whenever a request to /api is made, a matching route is looked up and it's GraphQL query is pulled up. This query is given to the graphql engine, the response is "RESTIFIED" and sent back.

In better words :)